modules/bullet/rigid_body_bullet.cpp

```diff
@@ -927,10 +928,10 @@ void RigidBodyBullet::reload_space_override_modificator() {
 		}
 
 		switch (currentArea->get_spOv_mode()) {
-			///case PhysicsServer::AREA_SPACE_OVERRIDE_DISABLED:
-			/// This area does not affect gravity/damp. These are generally areas
-			/// that exist only to detect collisions, and objects entering or exiting them.
-			///    break;
+			case PhysicsServer::AREA_SPACE_OVERRIDE_DISABLED:
+				/// This area does not affect gravity/damp. These are generally areas
+				/// that exist only to detect collisions, and objects entering or exiting them.
+				break;
```

Comment:
```
modules/bullet/rigid_body_bullet.cpp:929:10: warning: enumeration value 'AREA_SPACE_OVERRIDE_DISABLED' not handled in switch [-Wswitch]
```
@AndreaCatania

Reply:
Also here, it's useless check addition

Reply:
It's not useless, as it silences the warning + it shows that we took care of this possible branch and decided that nothing should be done for it (instead of forgetting about it, which could possibly mean a bug if we actually meant to do something with it).